### PR TITLE
Fix python script check_for_new_version.py to update version correctly

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

File: .github/scripts/check_for_new_version.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def validate_xml(file_name):
     if validate_xml(f"DROID_SignatureFile_V{latest_droid_version}.xml"):
         conf_replaced = (replace_line(line, 'droid', latest_droid_version, "\n") for line in lines)
     if validate_xml(f"container-signature-{latest_containers_version}.xml"):
-        conf_replaced = (replace_line(line, 'containers', latest_containers_version) for line in lines)
+        conf_replaced = (replace_line(line, 'containers', latest_containers_version) for line in conf_replaced)
```
Comment (TomJKing) [inline .github/scripts/check_for_new_version.py:69]: Will this return both the `DROID` and `Signature` versions in the `conf_replaced`?

**Review by TomJKing — COMMENTED** [review]
Looks good. Just one minor question.

File: .github/scripts/check_for_new_version.py (hunk 1 above)
Comment (vimleshtna) [reply .github/scripts/check_for_new_version.py:69]: Yes, I have tested that locally.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks.

**Pull request merged**